Kudu.Services/Deployment/DeploymentController.cs

```diff
@@ -111,8 +111,9 @@ public void Delete(string id)
                             {
                                 CreateDeployment(deployResult, result.Value<string>("details"));
 
-                                deployResult.Url = Request.RequestUri;
-                                deployResult.LogUrl = UriHelper.MakeRelative(Request.RequestUri, "log");
+                                Uri baseUri = UriHelper.MakeRelative(UriHelper.GetBaseUri(Request), Request.RequestUri.AbsolutePath);
```

Comment:
i think I should rename the variable to be deploymentUri to make it more clear. 

e.g if final url is 
````
https://kudutry.scm.azurewebsites.net/api/deployments/ef52ec67fc9574e726955a9cbaf7bcba791e4e95/log
````

baseUri is inted to be 
````
https://kudutry.scm.azurewebsites.net/api/deployments/ef52ec67fc9574e726955a9cbaf7bcba791e4e95
````